docs/presentation/Hierarchies/index.md

```diff
@@ -1,34 +1,46 @@
 # Hierarchies
 
-There are 2 primary concepts for creating hierarchies: rules and specifications.
+The Presentation library provides a declarative way to create hierarchies based on iModel data.
 
-## Rules
+![Hierarchy rule example](./media/hierarchy-rule-example.png)
 
-Define *where* and *if* specific branch should be created in the hierarchy. There are 2 types of rules:
+## Reference
 
-- [RootNodeRule](./RootNodeRule.md)
-- [ChildNodeRule](./ChildNodeRule.md)
+There are 2 primary concepts for creating hierarchies: [rules](#rules) and [specifications](#specifications).
 
-## Specifications
+### Rules
 
-Define *contents* for each branch. There are 6 types of specifications:
+Hierarchy presentation rule is a top level concept for defining hierarchies. There are two types of hierarchy rules:
 
-- [RelatedInstanceNodes](./RelatedInstanceNodes.md)
-- [InstanceNodesOfSpecificClasses](./InstanceNodesOfSpecificClasses.md)
-- [CustomQueryInstanceNodes](./CustomQueryInstanceNodes.md)
-- [CustomNode](./CustomNode.md)
+- [Root node rule](./RootNodeRule.md) is for creating nodes at the root level.
+- [Child node rule](./ChildNodeRule.md) is for creating child nodes.
 
-Multiple specifications can contribute to the same branch by specifying multiple
-specifications in a single rule or specifying multiple rules that match the same
-parent node.
+The only difference between those two rules is that we're looking for root node rules when creating root level nodes and for child node rules otherwise.
 
-**Note:**  grouping and sorting is done at specification level which
-means nodes generated from different specifications do not get grouped and sorted together.
+Both kinds of rules have two types of attributes - for defining [placement](#placement-attributes) and for defining [branch content](#branch-content-attributes). The latter attributes only get used if the rule does get used after evaluating the former.
 
-## Hierarchy Customization
+#### Placement Attributes
 
-All [general use customization rules](../Customization/index.md#rules) can be applied to hierarchies. In addition, there
-are some hierarchy-specific customization rules:
+Placement attributes define **if** the rule should be used and **where** the nodes appear if it is.
 
```

Comment:
I want to avoid the word `condition`, because it's an attribute name. Placement attributes include `condition`, but there are also other attributes that are taken into account when evaluating whether the rule should be used.